<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>950</width>
      <height>517</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Documents/AProg/gitlab/ToDo/src/Welcome_Window1/icons/background_Settings.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>852</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#666666;
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="all_task">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #F2F2F2;
</string>
    </property>
    <property name="text">
     <string>Number of all task:</string>
    </property>
   </widget>
   <widget class="QLabel" name="completted">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #F2F2F2;
</string>
    </property>
    <property name="text">
     <string>Number of all completted task:</string>
    </property>
   </widget>
   <widget class="QLabel" name="failed">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #F2F2F2;
</string>
    </property>
    <property name="text">
     <string>Number of all failed task:</string>
    </property>
   </widget>
   <widget class="QLabel" name="unfinished">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #F2F2F2;
</string>
    </property>
    <property name="text">
     <string>Number of all unfinished  task:</string>
    </property>
   </widget>
   <widget class="QLabel" name="number_all">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>286</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 24px;
color:#663366;
font-weight: bold;
</string>
    </property>
    <property name="text">
     <string>Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="completted_number">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>332</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #339900;
font-size: 24px;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="failed_number">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>387</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #990000;
font-weight: bold;
font-size:24px;
</string>
    </property>
    <property name="text">
     <string>Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="unfinished_number">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>427</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FFFF00;
font-weight: bold;
font-size:24px;
</string>
    </property>
    <property name="text">
     <string>Number</string>
    </property>
   </widget>
   <widget class="QToolButton" name="settings_but">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>10</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
